examples/sandbox/index.ts

```diff
@@ -15,6 +15,8 @@ import { WebUSBKeepKeyAdapter } from '@shapeshiftoss/hdwallet-keepkey-webusb'
 import { TCPKeepKeyAdapter } from '@shapeshiftoss/hdwallet-keepkey-tcp'
 import { TrezorAdapter } from '@shapeshiftoss/hdwallet-trezor-connect'
 import { WebUSBLedgerAdapter } from '@shapeshiftoss/hdwallet-ledger-webusb'
+import { Web3PortisAdapter } from '@shapeshiftoss/hdwallet-web3-portis'
```

Comment:
small nit `hdwallet-portis-web3`

Reply:
should be two packages: `hdwallet-web3` and `hdwallet-portis`. And I don't think we need a transport for it, since there's only ever going to be one way to talk to a Portis wallet.